src/main/config/config.c

```diff
@@ -657,6 +658,7 @@ static void resetConf(void)
     targetConfiguration();
 #endif
 
+    
```

Comment:
Yes it was accidentally,  I change something there when I first started to test this and then removed it. It was a mistake to commit that file.